tests/vmi_configuration_test.go

```diff
@@ -1160,7 +1220,12 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			var rngVmi *v1.VirtualMachineInstance
 
 			BeforeEach(func() {
-				rngVmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+				rngVmi = libvmi.New(
```

Comment:
Why you are not using `NewAlpine`?

Reply:
NewAlpine adds Rng by default, that would conflict with the second spec in this Context

Reply:
You could override it with an option, couldn't you?

Reply:
Adding a `NoRng` option? Perhaps you can do it here specifically, but I would not like to see a non common option like this in libvmi (unless you tell me it is common).

Reply:
@xpivarc Do you mean something like `WithoutRng`?

Reply:
Added